doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,90 @@
+================
+Cross Validation
+================
+
+Cross validation allows to estimate an algorithm's performance on unseen data.
+
+It is based on training the model on a subset of the data, and testing on the
+remaining data.
+
+The process is repeated for several complementary partitions and and all validation
+results are combined, e.g. by averaging.
+
+One type of cross validation is :math:`k-`fold cross validation.
+
```

Comment:
no paragraph break. Colon after sentence

Reply:
Fixed in the next commit

Reply:
Pls only comment if you have questions or comments. No need to say that you are changing this in the next commit, just send the commit :) (trying to keep the noise in my inbox down)